docs/guides/ignore-events.mdx

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
I would remove this page. I think the endpoint is simple enough that the endpoint page is enough. This does not add much.